support-metrics-client/src/test/java/io/confluent/support/metrics/collectors/BasicCollectorTest.java

```diff
@@ -58,8 +57,17 @@ public void testCollectMetrics() {
     assertTrue(unixTimeAtTestStart <= basicRecord.getTimestamp());
     assertTrue(basicRecord.getTimestamp() <= time.nowInUnixTime());
     assertEquals(AppInfoParser.getVersion(), basicRecord.getKafkaVersion());
-    assertEquals(Version.getVersion(), basicRecord.getConfluentPlatformVersion());
+    assertEquals("unknown", basicRecord.getConfluentPlatformVersion());
```

Comment:
looks like gradle test runs return "5.4.0-SNAPSHOT" value. Can we use `BasicCollector.cpVersion(AppInfoParser.getVersion())` here?

Reply:
That's a bit odd as it returned "unknown" locally (even after running `./gradlew jar` to generate `kafka-version.properties`). In any case, I did what you suggested.